stackslib/src/chainstate/stacks/block.rs

```diff
@@ -804,7 +853,8 @@ impl StacksMessageCodec for StacksMicroblock {
         let header: StacksMicroblockHeader = read_next(fd)?;
         let txs: Vec<StacksTransaction> = {
             let mut bound_read = BoundReader::from_reader(fd, MAX_MESSAGE_LEN as u64);
-            read_next(&mut bound_read)
+            // The latest epoch where StacksMicroblock exist is Epoch25
```

Comment:
What do microblocks have to do with the code here?  Also, you can use `read_next()` since you're using a `BoundReader`